### DISCUSSION TITLE
Development On 2.8 Is On Hold

### DISCUSSION THREAD

**Discussion by Mrbt0907** [CosmicHikari/Weather2-Remastered#100] [Announcements]
Good morning everyone! I have some unfortunate and fortunate news.

I am putting the update on hold. I did make significant progress with the update, and I am really close to completing the next indev for my discord members; however because of the reason below, I cannot continue the update until further notice.

The good news is we got ahold of the team thanks to Fartsy, and Coros will give us the red light or green light if he approves of the update.  We are currently waiting for Coro's response. The team also requested for this update to be put on hold, so this project will be put on hold until we get a response. I sincerely hope he likes what we managed to do so far.

If he says yes to our project, then I will be fully working on the updates whenever I am able to and post this mod on curseforge (If he allows me to), but if he says no, I am going to close this repository and eliminate all uploads.

If this repository does go down, I am grateful to spark a bit of the community to come back and I am grateful to make everyone's days better with this mod. Have a great day everyone.

**Comment by Desoroxxx** [comment]
I am scared...

**Reply by CosmicHikari** [reply]
Even if he says no it was still a lot of fun contributing to this repository. Made a few friends along the way too.

**Comment by Desoroxxx** [comment]
Any news on this?

**Reply by Mrbt0907** [reply]
Still no response. Coros will eventually finish the update before we get a response likely.

**Reply by Mrbt0907** [reply]
His business partner gave us the okay to continue. We still await for his direct answer, but his business partner did inform us that he is unlikely to say no, and told us that we can continue.
